apps/omg/test/support/crypto.ex

```diff
@@ -65,13 +65,13 @@ defmodule OMG.DevCrypto do
   @doc """
   Produces a stand-alone, 65 bytes long, signature for message of arbitrary length.
```

Comment:
update doc

Reply:
what about this :point_up: ?

Reply:
Oops I've updated other docs :man_facepalming: